.travis.yml

```diff
@@ -1,23 +1,11 @@
 language: bash
-dist: xenial
 
-services:
-    - docker
-
-addons:
-    apt:
-        update: true
 env:
     global:
-        - DOCKER_COMPOSE_VERSION=1.24.0
-        -   DOCKER_COMPOSE_URL="https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-$(uname -s)-$(uname -m)"
```

Comment:
Docker and Docker compose is installed by default according to https://docs.travis-ci.com/user/reference/xenial/#docker

:+1: